lib/pdf/reader/width_calculator/true_type.rb

```diff
@@ -10,8 +10,8 @@ class TrueType
       def initialize(font)
         @font = font
 
-        if @font.font_descriptor
-          @missing_width = @font.font_descriptor.missing_width
+        if fd = @font.font_descriptor
+          @missing_width = fd.missing_width
         else
           @missing_width = 0
         end
```

Comment:
This kind of syntax change is just to keep sorbet happy. We know that `@font.font_descriptor` will always be `nil` or never be `nil` so multiple calls to it are safe if the first is non-nil. That's not the case for all methods in ruby though, so sorbet pings us.